cogment_verse/specs/session_helper.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2023 AI Redefined Inc. <dev+cogment@ai-r.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import ABC, abstractmethod
+
+from ..constants import PLAYER_ACTOR_CLASS, TEACHER_ACTOR_CLASS, OBSERVER_ACTOR_CLASS, EVALUATOR_ACTOR_CLASS
+
+
+class PlayerAction:
```

Comment:
The methods in class `PlayerAction` could really benefit from small docstrings. Not sure it will be clear to everyone about what it is doing.

Reply:
yep

Reply:
Just pushed a bunch of documentation / type annotation